tests/decorate.ts

```diff
@@ -1,4 +1,4 @@
-import {
+import type {
```

Comment:
I was expecting this to be needed all over the place - I guess this was already changed in mod.ts some time ago 🤷 

(Thanks for updating!)

Reply:
@lucacasonato I should update the README's too? Right?

Reply:
Oh yeah. I missed those